test/evergreen.yml

```diff
@@ -4589,89 +4570,112 @@ tasks:
-  - <<: *format-stress-data-validation-test-disagg-follower
-    name: format-stress-data-validation-test-disagg-follower-3
+  - <<: *format-stress-test-disagg-follower
+    name: format-stress-test-disagg-follower-4
     tags: ["stress-test-disagg"]
+
+  # FIXME-WT-14566: Enable disagg switch mode on all platforms once failures have been resolved.
+  - <<: *format-stress-test-disagg-switch-palm
+    name: format-stress-test-disagg-switch-palm-1
+    tags: ["stress-test-disagg-fail", "stress-test-disagg-san-fail"]
+  - <<: *format-stress-test-disagg-switch-palm
+    name: format-stress-test-disagg-switch-palm-2
+    tags: ["stress-test-disagg-fail"]
+  - <<: *format-stress-test-disagg-switch-palm
+    name: format-stress-test-disagg-switch-palm-3
+    tags: ["stress-test-disagg-fail"]
+
   # FIXME-WT-14566: Enable disagg switch mode on all platforms once failures have been resolved.
   - <<: *format-stress-test-disagg-switch
     name: format-stress-test-disagg-switch-1
-    tags: ["stress-test-disagg-fail", "stress-test-disagg-san-fail"]
+    tags: ["stress-test-disagg-fail"]
   - <<: *format-stress-test-disagg-switch
     name: format-stress-test-disagg-switch-2
     tags: ["stress-test-disagg-fail"]
   - <<: *format-stress-test-disagg-switch
     name: format-stress-test-disagg-switch-3
     tags: ["stress-test-disagg-fail"]
 
-   # FIXME-WT-14566: Enable disagg leader mode on all platforms once failures have been resolved.
-  - <<: *format-stress-test-disagg-leader-palite
-    name: format-stress-test-disagg-leader-palite-1
-    tags: ["stress-test-disagg-fail"]
-  - <<: *format-stress-test-disagg-leader-palite
-    name: format-stress-test-disagg-leader-palite-2
-    tags: ["stress-test-disagg-fail"]
-  - <<: *format-stress-test-disagg-leader-palite
-    name: format-stress-test-disagg-leader-palite-3
-    tags: ["stress-test-disagg-fail"]
-  - <<: *format-stress-data-validation-test-disagg-leader-palite
-    name: format-stress-data-validation-test-disagg-leader-palite-1
-    tags: ["stress-test-disagg-fail"]
-  - <<: *format-stress-data-validation-test-disagg-leader-palite
-    name: format-stress-data-validation-test-disagg-leader-palite-2
-    tags: ["stress-test-disagg-fail"]
-  - <<: *format-stress-data-validation-test-disagg-leader-palite
-    name: format-stress-data-validation-test-disagg-leader-palite-3
-    tags: ["stress-test-disagg-fail"]
-  # FIXME-WT-14566: Enable disagg switch mode on all platforms once failures have been resolved.
-  - <<: *format-stress-test-disagg-switch-palite
-    name: format-stress-test-disagg-switch-palite-1
-    tags: ["stress-test-disagg-fail"]
-  - <<: *format-stress-test-disagg-switch-palite
-    name: format-stress-test-disagg-switch-palite-2
-    tags: ["stress-test-disagg-fail"]
-  - <<: *format-stress-test-disagg-switch-palite
-    name: format-stress-test-disagg-switch-palite-3
-    tags: ["stress-test-disagg-fail"]
+  - name: format-stress-disagg-leader-pull-request-test
```

Comment:
IMO, i'd rather us have a ticket directly enable and do this work. Otherwise its doing duplicate work as of other stress tests.

Reply:
Sorry I don't understand. The next ticket is going to fix this.